src/main/java/team03/mopl/domain/curation/controller/CurationController.java

```diff
@@ -33,13 +35,14 @@ public ResponseEntity<KeywordDto> registerKeyword(@Valid @RequestBody KeywordReq
   }
 
   @GetMapping("/{keywordId}/contents")
-  public ResponseEntity<List<ContentDto>> getRecommendations(
+  public ResponseEntity<CursorPageResponseDto<ContentDto>> getRecommendations(
       @PathVariable UUID keywordId,
+      @ModelAttribute CursorPageRequest request,
```

Comment:
```suggestion
      @ParameterObject @ModelAttribute CursorPageRequest request,
```
@ModelAttribute 사용해서 파라미터 받고있는데, @ModelAttribute 만 사용하면 Swagger에서 파라미터 설명 작성이 어렵고 Json 형태로 보내야하더라구요.  @ParameterObject  붙이면 일반 파라미터처럼 나타나는 것 같습니다!